client/my-sites/checkout/checkout/credit-card-selector.jsx

```diff
@@ -71,22 +54,7 @@ class CreditCardSelector extends React.Component {
 	}
 
 	newCardForm = () => {
-		const onSelect = () => this.handleClickedSection( 'new-card' );
-		const classes = classNames( 'checkout__payment-box-section', {
-			'no-stored-cards': this.props.cards.length === 0,
-		} );
-		const selected = 'new-card' === this.state.section;
-
-		return (
-			<CreditCard key="new-card" className={ classes } selected={ selected } onSelect={ onSelect }>
-				<NewCardForm
-					countriesList={ this.props.countriesList }
-					transaction={ this.props.transaction }
-					hasStoredCards={ this.props.cards.length > 0 }
-					selected={ selected }
-				/>
-			</CreditCard>
-		);
+		return null;
```

Comment:
Same here; since this method is internal to the component, you could remove it and whatever calls it without risk that you'll affect some other component.